src/shogun2-util/src/main/java/de/terrestris/shogun2/util/http/HttpUtil.java

```diff
@@ -486,6 +488,138 @@ private static Response postParams(HttpPost httpRequest,
 	}
 
 	/**
+	 * Perform HTTP PUT with empty body
+	 *
+	 * @param uri
+	 * @return
+	 * @throws HttpException
+	 * @throws URISyntaxException
+	 */
+	public static Response put(URI uri) throws URISyntaxException, HttpException {
+		return putBody(new HttpPut(uri), StringUtils.EMPTY, ContentType.APPLICATION_JSON, null, null);
```

Comment:
According to issue in method `putBody(...)` further down, will pass `null` here for `body` and `ContentType`.